packages/fxa-react/configs/tailwind.js

```diff
@@ -6,12 +6,8 @@ const plugin = require('tailwindcss/plugin');
 const tailwindcssDir = require('tailwindcss-dir');
 
 module.exports = {
-  purge: {
-    // The default here targets a standard CRA
-    // setup, but can be overridden as needed
-    content: ['./src/**/*.tsx', './public/index.html'],
-    whitelist: [],
-  },
+  // This targets a standard CRA setup, but can be overridden as needed
+  content: ['./src/**/*.tsx', './public/index.html'],
```

Comment:
Also interesting, but 1000% non-blocking for this PR&hellip; https://tailwindcss.com/blog/tailwindcss-v3-1#first-party-type-script-types

Not sure if we can just add the JSDoc-esque `@type {import('tailwindcss').Config}` here, or if it'd need to be attached to each `module.exports` statement in each tailwind.config.js file.